src/config.rs

```diff
@@ -116,6 +119,17 @@ impl<'a> Builder<'a> {
         }
     }
 
+    /// Set color driving voltage
+    pub fn yellow(self, yellow: &'a bool) -> Self {
```

Comment:
should this just be in some sort of setup instead of this function that is easily forgettable? I could make it an option to to `Builder::new`, or maybe create `.color` with an enum that accepts ::Red/::Yellow, and automatically populates the LUT with our defaults + writes that command to source the driving voltage if its yellow. What are your thoughts?